### PR TITLE
Ubuntu 22.04 backport to 20.04

### PR DESCRIPTION
**Pull request by cjdcordeiro**
Needs https://github.com/canonical/chisel-releases/pull/42.

## Contents

This PR backports all slice definition files (for the packages that are available in Focal) onto the Focal chisel release.

### Included in this PR

 - busybox
 - ca-certificates-java
 - fontconfig-config
 - fonts-dejavu-core
 - hello
 - libapr1
 - libblkid1
 - libbrotli1
 - libcom-err2
 - libcrypt1
 - libexpat1
 - libffi7
 - libfontconfig1
 - libfreetype6
 - libglib2.0-0
 - libgraphite2-3
 - libjpeg-turbo8
 - libk5crypto3
 - libkeyutils1
 - libkrb5-3
 - libkrb5support0
 - liblcms2-2
 - liblzma5
 - libmount1
 - libnspr4
 - libnss3
 - libnuma1
 - libpcre2-8-0
 - libpcre3
 - libpcsclite1
 - libpng16-16
 - libselinux1
 - libsqlite3-0
 - libssl1.1
 - libstdc++6
 - libunwind8
 - libuuid1
 - openjdk-8-jre-headless
 - openssl
 - zlib1g

From these, the following had to be modified:

 - libuuid1
 - libblkid1
 - libmount1
 - libglib2.0-0
 - libkrb5-3
 - openssl

Diff:
```diff
diff --git a/slices/libuuid1.yaml b/slices/libuuid1.yaml
index 5fa35e4..1cf7bd4 100644
--- a/slices/libuuid1.yaml
+++ b/slices/libuuid1.yaml
@@ -5,4 +5,4 @@ slices:
     essential:
       - libc6_libs
     contents:
-      /usr/lib/*-linux-*/libuuid.so.1*:
+      /lib/*-linux-*/libuuid.so.1*:
diff --git a/slices/libblkid1.yaml b/slices/libblkid1.yaml
index a24f261..f51501b 100644
--- a/slices/libblkid1.yaml
+++ b/slices/libblkid1.yaml
@@ -5,4 +5,4 @@ slices:
     essential:
       - libc6_libs
     contents:
-      /usr/lib/*-linux-*/libblkid.so.1*:
+      /lib/*-linux-*/libblkid.so.1*:
diff --git a/slices/libmount1.yaml b/slices/libmount1.yaml
index 596c45a..800bed9 100644
--- a/slices/libmount1.yaml
+++ b/slices/libmount1.yaml
@@ -7,4 +7,4 @@ slices:
       - libselinux1_libs
       - libblkid1_libs
     contents:
-      /usr/lib/*-linux-*/libmount.so.1*:
+      /lib/*-linux-*/libmount.so.1*:
diff --git a/slices/libglib2.0-0.yaml b/slices/libglib2.0-0.yaml
index b5322c9..6507f3a 100644
--- a/slices/libglib2.0-0.yaml
+++ b/slices/libglib2.0-0.yaml
@@ -11,7 +11,7 @@ slices:
   libs:
     essential:
       - libglib2.0-0_core
-      - libffi8_libs
+      - libffi7_libs
       - libmount1_libs
       - libselinux1_libs
       - zlib1g_libs
diff --git a/slices/libkrb5-3.yaml b/slices/libkrb5-3.yaml
index cae9c80..1be2a0b 100644
--- a/slices/libkrb5-3.yaml
+++ b/slices/libkrb5-3.yaml
@@ -8,7 +8,7 @@ slices:
       - libk5crypto3_libs
       - libkeyutils1_libs
       - libkrb5support0_libs
-      - libssl3_libs
+      - libssl1.1_libs
     contents:
       /usr/lib/*-linux-*/krb5/plugins/libkrb5/:
       /usr/lib/*-linux-*/krb5/plugins/preauth/spake.so:
diff --git a/slices/openssl.yaml b/slices/openssl.yaml
index 218c509..dfc2c9b 100644
--- a/slices/openssl.yaml
+++ b/slices/openssl.yaml
@@ -5,7 +5,7 @@ slices:
     essential:
       - libc6_libs
       - libc6_config
-      - libssl3_libs
+      - libssl1.1_libs
       - openssl_config
     contents:
       /usr/bin/c_rehash:
```

And the following are new (ie. they don't exist in Jammy but were needed as dependencies for the above list):

 - libffi7
 - libssl1.1

### CODE REVIEW

File: slices/libffi7.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package: libffi7
+
+slices:
+  libs:
+    essential:
+      - libc6_libs
+    contents:
+      /usr/lib/*-linux-*/libffi.so.7*:
```
Comment (rebornplusplus) [inline slices/libffi7.yaml:8]: nitpicking: no end-of-line after the last line.

**Review by rebornplusplus — APPROVED** [review]
Looks good to me. Nice work there!

File: slices/libffi7.yaml (hunk 1 above)
Comment (cjdcordeiro) [reply slices/libffi7.yaml]: Thanks ;) done

**Comment by woky** [comment]
Please rebase on top of the dependent (and now merged) PR. Currently, this doesn't work (missing libc6).

**Comment by cjdcordeiro** [comment]
> Please rebase on top of the dependent (and now merged) PR. Currently, this doesn't work (missing libc6).

Done

**Review by woky — APPROVED** [review]
LGTM

**Pull request merged**